metecho/api/views.py

```diff
@@ -191,6 +199,18 @@ class ProjectViewSet(CreatePrMixin, ModelViewSet):
     filterset_class = ProjectFilter
     error_pr_exists = _("Project has already been submitted for testing.")
 
+    def get_queryset(self):
+        qs = super().get_queryset()
+        whens = [
+            When(status=PROJECT_STATUSES.Review, then=0),
+            When(status=PROJECT_STATUSES["In progress"], then=1),
+            When(status=PROJECT_STATUSES.Planned, then=2),
+            When(status=PROJECT_STATUSES.Merged, then=3),
+        ]
+        return qs.annotate(ordering=Case(*whens, output_field=IntegerField())).order_by(
+            "ordering"
+        )
```

Comment:
@wlonk If `ordering` is the same between two models, it looks like it's no longer applying the secondary sort of `-created_at`.